pom.xml

```diff
@@ -5,34 +5,43 @@
     <parent>
         <groupId>org.cyclopsgroup</groupId>
         <artifactId>cyclopsgroup-java-parent</artifactId>
-        <version>0.6</version>
+        <version>0.6.1</version>
     </parent>
     <artifactId>caff</artifactId>
     <name>caff</name>
     <version>0.4.1-SNAPSHOT</version>
     <description>Java conversion and formatting framework</description>
     <url>http://www.cyclopsgroup.org/projects/caff</url>
+    <properties>
+        <jdk.source.version>1.7</jdk.source.version>
+        <jdk.target.version>1.7</jdk.target.version>
```

Comment:
Does the new jdk target version break the latest Jmxterm?

Reply:
The latest Jmxterm (1.0.0) uses an older version of caff (3.0). I'm currently committing some more pull requests, that update the JDK versions to a least 1.7. I'm trying to let you decide when to update the cyclopsgroup dependencies.